src/main/resources/plugin.yml

```diff
@@ -82,4 +82,12 @@ commands:
   generate:
     aliases: [worlds, gen]
     description: Pre-generate worlds
-    usage: /<command> stop | <start> <numWorlds>
\ No newline at end of file
+    usage: /<command> stop | <start> <numWorlds>
+
+  tpt:
```

Comment:
I prefer to have the base command names be a bit more self-explanatory. So perhaps call it `teamteleport` and alias it to whatever is easy to type and use.